### PR TITLE
Added articles and Changed wordings (fixes #2711)

### LINKED ISSUE #2711: Missing articles and redundancy on Nation Planet page
### Problem
There should be a 'an' preceding Administrator and a 'a' preceding whole. Also, the phrase 'a number of' can be changed to a shorter word for conciseness.

### Steps to reproduce the problem

Go to Step 7: Nation Planet tab and before the title "Check Sync Status", the paragraph with the missing articles is situated.

### Screenshots

Screenshot (31)

### Proposed solution

Add 'an' before Administrator, a before whole and change 'a number of ' to several.

### PR DESCRIPTION
**Pull request by smitshrestha101**
Fixes #2711 

### Description
Added articles to multiple words that were missing articles and changed some wordings to make it more concise.

### Raw.Githack preview link

(https://raw.githack.com/smitshrestha101/smitshrestha101.github.io/update-nation-tab/#!pages/vi/vi-nation.md)

### CODE REVIEW

File: pages/vi/vi-nation.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 ## Objectives
 
-* Learn to sync your Community with the Nation
-* Use the Nation interface to check whether your Community successfully joined the Nation
-* Update your Community Planet to the latest version
+- Learn to sync your Community with the Nation
```
Comment (ChiragAwale) [inline pages/vi/vi-nation.md]: Are these intentional?

File: pages/vi/vi-nation.md [hunk 2]
```diff
@@ -16,10 +16,11 @@ Make sure vagrant is running and then click [here](http://localhost:3100) to acc
 It is recommended that you use Firefox since Planet is only guaranteed to run properly on Firefox.
 
 **NOTES**: After you register your community, but before you can sync with the nation, you need to create an additional dummy user on your community. Therefore, to create a dummy user:
-1. Create a quick additional user under "Become a Member" on the login page (HINT: When creating the dummy user, don't give it   a password that you actually use).
+
```
Comment (ChiragAwale) [inline pages/vi/vi-nation.md]: This too?

File: pages/vi/vi-nation.md (hunk 1 above)
Comment (lmmrssa) [reply pages/vi/vi-nation.md]: @smitshrestha101 This change is not required as `*` also does generate list. So before making changes please also check if it is having any issue or not

**Comment by smitshrestha101** [comment]
I resolved all the issues

(https://raw.githack.com/smitshrestha101/smitshrestha101.github.io/update-nation-tab/#!pages/vi/vi-nation.md)

This should do it. Thank you!

**Comment by lmmrssa** [comment]
This is not any change just one whitespace removed. So closing this PR

**Comment by ChiragAwale** [comment]
@smitshrestha101

**Pull request closed without merging**